query/src/sessions/session_settings.rs

```diff
@@ -110,10 +110,10 @@ impl Settings {
                 desc: "Enable planner v2 by setting this variable to 1, default value: 0",
             },
             SettingValue {
-                default_value: DataValue::String("\n".as_bytes().to_vec()),
-                user_setting: UserSetting::create("record_delimiter", DataValue::String("\n".as_bytes().to_vec())),
+                default_value: DataValue::String(" ".as_bytes().to_vec()),
+                user_setting: UserSetting::create("record_delimiter", DataValue::String(" ".as_bytes().to_vec())),
                 level: ScopeLevel::Session,
-                desc: "Format record_delimiter, default value: \n",
+                desc: "Format record_delimiter, default value:  ",
```

Comment:
We do use `\n` as `record_delimiter`. Changing it into space is wrong.

I prefer to escape it while in displaying

```suggestion
                desc: "Format record_delimiter, default value: \\n",
```

Reply:
I got it, this default value of this field is `\n` itself. 
I will fix it. Thank you for your review.

Reply:
Now we escaped `record_delimiter` from "\n"  to be "\\n", but it will make `record_delimiter ` not work. Because some codes depend on this settings, like [format_csv](https://github.com/datafuselabs/databend/blob/3b34c0f6a79bfaa2dedfc3622f09d7aa73902163/query/src/formats/format_csv.rs#L88)

Reply:
> Now we escaped `record_delimiter` from "\n" to be "\n", but it will make `record_delimiter ` not work. Because some codes depend on this settings, like [format_csv](https://github.com/datafuselabs/databend/blob/3b34c0f6a79bfaa2dedfc3622f09d7aa73902163/query/src/formats/format_csv.rs#L88)

Got it, so I will close this pr and read the code carefully.